torchci/pages/flakytest.tsx

```diff
@@ -20,71 +21,42 @@ export default function Page() {
 
   return (
     <div>
-      <h1>PyTorch CI Flaky Tests</h1>
+      <h1>PyTorch CI Test Failures and Flaky Tests</h1>
+      <div>
+        This shows the most recent 100 failures in CI from after March 22nd,
+        2023. Data prior to this date still exists, but can only be obtained by
```

Comment:
Nit: 2024 :)

Reply:
Its referencing a change from last year, i added a link to it in the code

Reply:
Ohh, I see, I thought it was yesterday `Jan 22nd` lol

Reply:
Although I think the date could go, I don't think if people care about data more than a year ago, but who knows.